### PR TITLE
Typo in snapcast integration

### PR DESCRIPTION
**Pull request by BarrettLowe**
## Proposed change

Typo - snapcast service arguments are required for set_latency.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/snapcast.markdown [hunk 1]
```diff
@@ -74,5 +74,5 @@ Set the latency of a speaker.
 
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
-| `entity_id` | yes | String or list of `entity_id`s for which latency will be adjusted.
-| `latency` | yes | Latency in ms.
+| `entity_id` | no | String or list of `entity_id`s for which latency will be adjusted.
```
Comment (frenck) [inline source/_integrations/snapcast.markdown]: `entity_id` is optional, so please revert the change on this line.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by BarrettLowe** [comment]
Hm. You are correct. It SHOULD be so now there is a pull request in core that makes it requires. I have added the core PR link to this PR and vise-versa.

File: source/_integrations/snapcast.markdown (hunk 1 above)
Comment (MartinHjelmare) [reply source/_integrations/snapcast.markdown]: Either entity_id or area_id is required.

**Comment by BarrettLowe** [comment]
Want me to rebase this on 'next' since now I've added code to core also? That service wasn't written to have entity_id ignored so I feel like it should go in the documentation now... Please advise. Thanks

File: source/_integrations/snapcast.markdown (hunk 1 above)
Comment (BarrettLowe) [reply source/_integrations/snapcast.markdown]: I'm a bit confused on what to do now. Should I add add 'area_id' an make both 'entity_id' and 'area_id' optional? Just want to make sure it's clear. Thanks

Comment (frenck) [reply source/_integrations/snapcast.markdown]: We currently do not document the use of area_id.
A) General documentation around it is missing
B) It is not really useable manually (finding the area ID to use is hard).

I've adjusted the PR to correct the entity_id.

**Review by frenck — APPROVED** [review]
